src/VisualStudio/Core/Def/Implementation/CodeCleanup/CodeCleanUpFixer.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.Language.CodeCleanUp;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.CodeCleanup
+{
+    internal abstract class CodeCleanUpFixer : ICodeCleanUpFixer
+    {
+        public abstract Task<bool> FixAsync(ICodeCleanUpScope scope, FixIdContainer enabledFixIds, CancellationToken cancellationToken);
```

Comment:
note sure what purpose this class services currently...

Reply:
this is the type we MEF imports to report to editor + content type.

Reply:
➡️ Documented this in 44c572ce4b940394d290681816fb3143dc65ad1d